H5P.BranchedVideo/branched-video.js

```diff
@@ -613,39 +636,132 @@ H5P.BranchedVideo = (function ($) {
       volumeSlider.oninput = function(){
         var currentVid = getBranch(currentVideoPlaying).getVideoHTML();
         var volume = volumeSlider.value / 100;
         currentVid.volume = volume;
+        var tempVal = volumeSlider.value / volumeSlider.max;
+        $('.tapestry-volume-slider').css({'background-image':
+            '-webkit-gradient(linear, left top, right top, '
+            + 'color-stop(' + tempVal + ', #FFFFFF), '
+            + 'color-stop(' + tempVal + ', #606060))'
+          });
       }
+
       volumeDiv.appendChild(volumeSlider);
       volumeDiv.appendChild(volumeButton);
 
+
+      // TODO: update
       // SETTINGS
       var settingsButton = document.createElement('button');
       settingsButton.type = 'button';
       settingsButton.className = 'tapestry-settings-button';
-      var helpNode = document.createElement('h');
-      var helpNodeText = document.createTextNode('help mode enabled');
-      helpNode.className = 'tapestry-time-text';
-      helpNode.style.display = 'none';
-      helpNode.style.position = 'absolute';
-      helpNode.style.top = '90%';
-      helpNode.style.left = '30%';
-      helpNode.appendChild(helpNodeText);
-      rightControls.appendChild(helpNode);
-
-      // functions for handling onhover help button
+      var settingsDiv = document.createElement('div');
+      settingsDiv.className = 'tapestry-settings-dropdown-content';
+      settingsDiv.style.display = 'none';
+      rightControls.appendChild(settingsDiv);
+
+      // help mode
+      var helpButton = document.createElement('button');
+      helpButton.type = 'button';
+      helpButton.style.position = 'absolute';
+      helpButton.style.fontSize = '10px';
+      helpButton.style.width = '100px';
+      helpButton.style.backgroundColor = 'white';
+      var helpButtonText = document.createTextNode('Help Mode');
+      helpButton.appendChild(helpButtonText);
+      settingsDiv.appendChild(helpButton);
+
+      helpButton.onclick = function(){
+        if (self.helpMode == false){
+          helpButton.innerHTML = 'Help Mode &#10003';
+          self.helpMode = true;
+        } else {
+          helpButton.innerHTML = 'Help Mode';
+          self.helpMode = false;
+        }
+      }
+
+      helpButton.onmouseenter = function(){
+        helpButton.style.backgroundColor = '#3f89ff';
+        helpButton.style.color = 'white';
+      }
+      helpButton.onmouseleave = function(){
+        helpButton.style.backgroundColor = 'white';
+        helpButton.style.color = 'black';
+      }
+
+      // closed caption
+      var ccButton = document.createElement('button');
+      ccButton.style.position = 'absolute';
+      ccButton.style.top = '15px';
+      ccButton.style.fontSize = '10px';
+      ccButton.style.width = '100px';
+      ccButton.type = 'button';
+      var ccButtonText = document.createTextNode('Closed Caption');
+      ccButton.appendChild(ccButtonText);
+      settingsDiv.appendChild(ccButton);
+
+
```

Comment:
Please watch for your spacing. Try not to have 2 line breaks unless there's a specific reason.